stripe_official.php

```diff
@@ -1444,6 +1444,8 @@ public function getPaymentMethods()
         $query->from('configuration');
         $query->where('name LIKE "STRIPE_PAYMENT%"');
         $query->where('value = "on"');
+        $query->where('id_shop_group = ' . $this->context->shop->id_shop_group);
```

Comment:
In case of new PrestaShop with only one shop, ps_configuration has an id_shop and id_shop_group = null

I think we should use 

Configuration::getValues([Stripe_Official::PAYMENT_VISA, Stripe_Official::PAYMENT_MASTERCARD, ...]);